app/controllers/expenses_controller.rb

```diff
@@ -0,0 +1,27 @@
+class ExpensesController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @expense = Expense.new
+    @categories = Category.all
```

Comment:
- [ ] Please make sure to solve the N+1 query problem as per the project requirements, consider using the includes method in order to do so.